jerry-core/config.h

```diff
@@ -17,6 +17,33 @@
 #define CONFIG_H
 
 /**
+ * Group of builtin-related features that can be disabled together.
+ */
+#ifdef CONFIG_DISABLE_BUILTINS
+# define CONFIG_DISABLE_ANNEXB_BUILTIN
```

Comment:
wrong style: no space needed after `#`

Reply:
The rest of this header uses this type of indentation. I did not want to mix styles within a single file so went for the space.

Reply:
Legacy... You can update those line too if you want.

Reply:
Just to stir things up a bit, why do we hate indentation in preproc conditions in the first place? Personally I would always prefer a well indented block over a giant blob of unindented mess anytime.

Reply:
We have 7 header + 3 source files that carry this legacy. Can I suggest to have this file in a uniform style for now and have a follow up style fix PR that does nothing else but gets rid of all the legacy hashmark-spaces everywhere?

Reply:
OK, let's land it this way and discuss the spaces later.